src/pages/Settings/TwoStepVerificationPage.vue

```diff
@@ -38,33 +93,143 @@
 </template>
 
 <script>
+import gql from 'graphql-tag';
 import KvIcon from '@/components/Kv/KvIcon';
 import KvSettingsCard from '@/components/Kv/KvSettingsCard';
+import KvButton from '@/components/Kv/KvButton';
 import TheMyKivaSecondaryMenu from '@/components/WwwFrame/Menus/TheMyKivaSecondaryMenu';
 import WwwPage from '@/components/WwwFrame/WwwPage';
+import removeMfa from '@/graphql/mutation/removeMfa.graphql';
+
+const mfaQuery = gql`query mfaQuery($mfa_token: String!) {
+	my {
+		lastLoginTimestamp @client
+		authenticatorEnrollments(mfa_token: $mfa_token) {
+			id
+			active
+			authenticator_type
+		}
+	}
+}`;
 
 export default {
+	data() {
+		return {
+			isMfaActive: false,
+			lastLoginTime: 0,
+		};
+	},
 	components: {
 		KvIcon,
 		KvSettingsCard,
+		KvButton,
 		TheMyKivaSecondaryMenu,
 		WwwPage,
 	},
 	metaInfo: {
 		title: '2-step verification',
 	},
+	mounted() {
+		if (this.kvAuth0.enabled) {
+			this.kvAuth0.checkSession();
+			this.kvAuth0.getMfaManagementToken()
```

Comment:
You need to wait for the `checkSession` promise to complete before calling `getMfaManagementToken`. You can do that by changing these lines to
```
this.kvAuth0.checkSession()
	.then(() => this.kvAuth0.getMfaManagementToken())
	.then(token => {
		...
```